homework/vector-of-shared-ptrs/vectorFunctions.hpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#include <vector>
+#include <memory>
+
+std::vector<std::shared_ptr<int>> generate(int count);
+void print(std::vector<std::shared_ptr<int>>temp_vector);

```

Comment:
```cpp
void print(std::vector<std::shared_ptr<int>> temp_vector);
```
Always put a space between the type and variable name. You can only skip when you use template types, but it is not consistent. Otherwise your code will never compile if you write something like
```cpp
std::vector<std::shared_ptr<int>> generate(intcount);
```